src/Microsoft.AspNet.Razor/CodeGenerators/Visitors/CSharpTagHelperRunnerInitializationVisitor.cs

```diff
@@ -26,6 +26,25 @@ public class CSharpTagHelperRunnerInitializationVisitor : CodeVisitor<CSharpCode
             _tagHelperContext = Context.Host.GeneratedClassContext.GeneratedTagHelperContext;
         }
 
+        /// <inheritdoc />
+        public override void Accept(Chunk chunk)
```

Comment:
That wouldn't work at all. `Visit(ParentChunk)` is called only if the `chunk` isn't one of the many subclasses of `ParentChunk`. This is exactly why the comments below say what they do.